index.html

```diff
@@ -1,53 +1,124 @@
-        <a href="https://dafny.org/dafny/DafnyRef/DafnyRef#51-specification-clauses">pre- and post-conditions,
-        termination conditions, loop invariants, and read/write specifications</a>.
-    </p>
-
-    <h2>Get Started</h2>
-    <p>
-        Try it out by installing the Visual Studio Code <a href="https://marketplace.visualstudio.com/items?itemName=dafny-lang.ide-vscode">extension</a>, which installs all dependencies apart from .NET 6.
-        To get a better sense of Dafny's capabilities, follow the <a href="https://dafny.org/dafny/OnlineTutorial/guide">tutorial</a>.
-    </p>
-
-    <h2>Get Involved</h2>
-    <p>
-        Development happens on <a href="https://github.com/dafny-lang/dafny">GitHub</a>.
-    </p>
+
+<div>
+<h1>Dafny</h1>
+</div>
+
+<table style="width:100%">
+<tr>
+<td style="width:15%">
+<img src="Banner/dafny-logo.jpg" alt="Dafny" width="180" height="150">
+</td>
+<td style="width:45%">
+<p>
+<b>Dafny</b> is a verification-aware programming language that has native support for recording specifications and is equipped with a static program verifier. By blending sophisticated automated reasoning with familiar programming idioms and tools, Dafny empowers developers to write provably correct code (w.r.t. specifications),
+while also compiling Dafny code to familiar development environments such as C#, Java, JavaScript and Go (with more in progress, such as Python) so Dafny can integrate with your existing workflow.
+Dafny makes rigorous verification an integral part of development thus reducing costly late-stage bugs that may be missed by testing.
+</p>
+</td>
+<td style="width:40%">
+<b>Quick Links:</b>
+<ul>
+<li> <a href="https://github.com/dafny-lang/dafny/wiki/INSTALL"><b>Installation</b></a> 
+     (or a <a href="https://marketplace.visualstudio.com/items?itemName=dafny-lang.ide-vscode">VSCode plugin for Dafny</a>)
+<li> <a href="https://dafny.org/dafny/DafnyRef/DafnyRef">Dafny Reference Manual and User Guide</a>
+<li> <a href="https://dafny.org/dafny/toc">Dafny Resources for Users</a>
+<li> <a href="https://github.com/dafny-lang/dafny">Dafny GitHub project (for developers of the Dafny tools themselves)</a>
+</ul>
+</td>
+
+
+</tr>
+</table>
+
+<table style="width:100%">
+<td style="width:65%" valign=top>
+<p>
+In addition to a verification engine to check implementation against specifications, the Dafny ecosystem includes several compilers, plugins for common software development IDEs, a LSP-based Language Server, a code formatter, a reference manual, tutorials, power user tips, books, the experiences of professors teaching Dafny, and the accumulating expertise of industrial projects using Dafny.
+Example Dafny code in a Visual Studio IDE is shown to the right.
+
+</p>
+
+<p>Dafny has support for common programming concepts such as </p>
+<ul>
+<li> mathematical and bounded integers and reals, bit-vectors, classes, iterators, arrays, tuples, generic types, refinement and inheritance
```

Comment:
```suggestion
<li> mathematical and bounded integers and reals, bit-vectors, classes, iterators, arrays, tuples, generic types, refinement and inheritance,
```

(Although perhaps this should be a two-level nested list using semicolons instead)

Reply:
Done